src/app/shared/components/inline-value-edit/inline-value-edit.component.spec.ts

```diff
@@ -357,4 +385,20 @@ describe('InlineValueEditComponent', () => {
     expect(Number(utcDt.format('D'))).toEqual(13);
     expect(Number(utcDt.format('h'))).toEqual(3);
   });
+
+  it('should expand the title when hovering with the mouse over it', () => {
+    component.type = 'text';
+    component.isPublicArchive = true;
+    fixture.detectChanges();
+
+    const nameContainer = fixture.debugElement.query(
+      By.css('.inline-value-text')
+    ).nativeElement;
+    nameContainer.dispatchEvent(new Event('mouseenter'));
+    fixture.detectChanges();
+
+    fixture.whenStable().then(() => {
+      expect(nameContainer.classList).toContain('is-name-hovered');
+    });
```

Comment:
Just for consistency can you make this test `async` and use `await fixture.whenStable()` instead?